autoload/cmake4vim.vim

```diff
@@ -170,7 +170,7 @@ function! cmake4vim#CTest(bang, ...) abort
     silent exec 'cd' l:build_dir
     let l:cmd = printf('ctest %s %s', a:bang ? '' : g:cmake_ctest_args, join( a:000 ) )
     " Run
-    call utils#common#executeCommand(l:cmd)
+    call utils#common#executeCommand(l:cmd, 1)
```

Comment:
Would it be possible just to call `copen` after this line, instead of changing `executeCommand`?

Reply:
It won't work, because `executeCommand` supports async modes (`job`).